api/v1.inc

```diff
@@ -37,6 +39,12 @@ $router->add_route("GET", "v1/projects/:projectid/pages", "api_v1_project_pages"
 $router->add_route("GET", "v1/projects/:projectid/pagedetails", "api_v1_project_pagedetails");
 $router->add_route("GET", "v1/projects/:projectid/pages/:pagename/pagerounds/:pageroundid", "api_v1_project_page_round");
 
+$router->add_route("GET", "v1/queues/rounds/:roundid", "api_v1_queues_rounds");
```

Comment:
Now that we have a unique ID for each queue we can treat them more like objects with round_id and name rather than only unique by (round_id, name) pairs. So what this means is that we can now have
`v1/queues` to return all of the queues (which include id, roundid, and name) and pass in `roundid` as a selection parameter to just get the queues for a specific round (v1/queues?roundid=P1). This then mirrors what we do for projects. This should be pretty easy to support by making `round_id` in the fetch function default to `null` and then just not filtering by roundid when null _but_ in all cases sorting by `round_id, ordering` instead of just `ordering`. It does mean that we need to keep track of when the results switch from one round to another to reset the group to null.

Also to mirror the project endpoints the function names should be massaged to:
```php
$router->add_route("GET", "v1/queues", "api_v1_queues");
$router->add_route("GET", "v1/queues/:queueid", "api_v1_queue");  # singular queue
$router->add_route("GET", "v1/queues/:queueid/stats", "api_v1_queue_stats");  # singular queue
$router->add_route("GET", "v1/queues/:queueid/projects", "api_v1_queue_projects");  # singular queue
```

Reply:
It's good that we've gone through a few iterations on this. We've ended up with a much better API.

Done.